Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
```

Comment:
File checking should be used

Reply:
Try catch handles FileNotFound exception

Reply:
yes but you can have more exceptions and if you print out general messages user/developer would not understand why program crashed